File: URI/2864.py
from math import pow

N = int(input())

for i in range(N):
	a,b,c = list(map(int,input().split()))

	delta = pow(b,2) - 4 * a * c
	result = -delta / (4.0*a)

	print("{:.2f}".format(result))

File: URI/C++/2864.cpp
#include<iostream>
#include<cmath>
using namespace std;

int main(){
	int N,a,b,c;
	double delta, result;
	cin >> N;

	for(int i=0; i<N; i++){
		cin >> a >> b >> c;
		delta = pow(b,2) - 4 * a * c;
		result = -delta / (4.0 * a);

		cout.precision(2);
		cout.setf(ios::fixed);
		cout << result << endl;
	}

	return 0;
}